test/league_statistics_test.rb

```diff
@@ -0,0 +1,145 @@
+require "./lib/league_statistics"
+require "minitest/autorun"
+require "minitest/pride"
+
+class LeagueStatisticsTest < MiniTest::Test
+
+  def setup
```

Comment:
setup and test it exists with attributes are the same across all inheriting classes. But we still have to test it for each specific class.